<launch>
<!-- start the bridge -->
   <!--param name="RIB_server_ip" value="111.111.111.111" type="str"/-->
   <!--param name="RIB_port" value="18944" type="int"/-->
   <!--param name="RIB_type" value="client" type="str"/-->
   <node pkg="ros_igtl_bridge" type="ros_igtl_bridge_node" name="ros_igtl_bridge_node" output="screen">

   </node>
</launch>

